packages/gateway-trpc/src/crypto.ts

```diff
@@ -0,0 +1,26 @@
+import { createRemoteJWKSet, jwtVerify } from "jose"
+
+export class JwtService {
+  private jwks: ReturnType<typeof createRemoteJWKSet> | null = null
```

Comment:
```suggestion
  private jwks: ReturnType<typeof createRemoteJWKSet> | null = null
```
There has to be a type to use here without doing TS-magic?